custom-fonts.md

```diff
@@ -23,6 +29,22 @@ flutter:
         - asset: assets/fonts/abrilfatface/AbrilFatface-Regular.ttf
 ```
 
+Second use the font by delcaring the `TextStyle` constant and adding the font family used in the `pubsec.yaml`.
+```
+// declare the text style
+const textStyle = const TextStyle(
+  fontFamily: 'Raleway',
+);
+
+// use the text style
+var buttonText = const Text(
+  "Use the font for this text",
+  style: textStyle,
+);
+```
+
+## pubspec.yaml spec breakdown
```

Comment:
Not sure if "breakdown" will be easy to understand for everyone :)

Maybe: Details ?